benchmarks/discrete_trajectory.cpp

```diff
@@ -79,7 +79,7 @@ DiscreteTrajectory<World> MakeTrajectory(Timeline<World> const& timeline,
 
 // Constructs a trajectory beginning with many empty segments.
 DiscreteTrajectory<World> MakeTrajectoryWithEmptySegments(
-    int num_empty_segments) {
+    int const num_empty_segments) {
```

Comment:
One last nit: we don't use `num_`.  We would either write `number_of_empty_segments` or just `empty_segments`.  Pick what you prefer.

Reply:
Done.